src/SFML/Window/Win32/InputImpl.cpp

```diff
@@ -512,6 +512,38 @@ UINT sfScanToVirtualKey(sf::Keyboard::Scancode code)
     // clang-format on
 }
 
+const char* sfScanToConsumerKeyName(sf::Keyboard::Scancode code)
```

Comment:
Would it be more expressive to return an optional? Perhaps `std::optional<std::string_view>`? If I see an API return a `const char*` I'm not automatically going to assume it may return a null pointer.